DeploymentCloud/Deployment.Common/Resources/Parameters/Resource-Parameter.json

```diff
@@ -103,6 +103,15 @@
         },
         "clusterName": {
             "value": "$serviceFabricName"
-        }
+        },
+		"sparkType": {
+			"value": "$sparkType"
+		},
+		"storageAccounts_spark_name": {
+			"value": "$sparkBlobAccountName"
+		},
+		"sparkManagedIdentity": {
```

Comment:
let's move this back to Spark-Template and Spark-Parameter. And add a logic to set KV policy as appropriate.

Reply:
done